functions/src/modules/SQR.functions.ts

```diff
@@ -4,6 +4,7 @@ import * as  admin from 'firebase-admin';
 const bucket = admin.storage().bucket();
 const db = admin.database();
 import * as helpers from './../helpers';
+import * as emailHelper from './email.functions';
 
 let emailTemplates = {};
```

Comment:
The reason for this global variable is unclear. I cannot see any code adding to this object.